### DISCUSSION TITLE
How to get pix2pix results that are as good as the pretrained models?

### DISCUSSION THREAD

**Discussion by mccallion** [junyanz/pytorch-CycleGAN-and-pix2pix#1333] [Q&A] [answered]
Hello! I ran the code in pix2pix.ipynb for the facades data and noticed that my results were much worse than results from the pretrained model's. Has anybody been able to replicate the image quality of the pretrained models, and if so, how did they do it? (Ideally the authors will chime in, but I'd love to hear from anybody!).

Thank you!

**Comment by mccallion** [comment, marked as answer]
Found the answer – I wasn't using the author's suggested params after all! Instead of their batch size = 1, I was using batch size = 16 and getting worse results. Using all of their code out of the box without modifications worked.

**Reply by TimmyYun** [reply]
Hello! So what parameters did you finally use to achieve good results with batch size 16 ? 
Thanks in advance

**Reply by mccallion** [reply]
I was not able to make it work with bs=16. I ended up using bs=1 and all of the other parameter values suggested by the authors.
